web_external/templates/body/featuresetPage.jade

```diff
@@ -1,54 +1,47 @@
-.isic-featureset-version-container
-  h4 Version
-  .isic-listing-content.isic-featureset-version-content
-   = featureset.get('version')
+- var user = featureset.get('creator')
 
-.isic-featureset-creator-container
-  h4 Creator
-  .isic-listing-content.isic-featureset-creator-content
-   - var user = featureset.get('creator')
-   | #{user.firstName} #{user.lastName} (#{user.login})
+.isic-featureset-container
+  table.table.table-condensed.isic-featureset-table
+    tr
+      td.isic-featureset-table-section-header(colspan=2) Info
+    tr
+      td.isic-featureset-table-label Version
+      td= featureset.get('version')
+    tr
+      td.isic-featureset-table-label Creator
+      td
+        | #{user.firstName} #{user.lastName} (#{user.login})
 
-.isic-featureset-global-features-container
-  h4 Global Features
-  .isic-listing-content.isic-featureset-global-features-content
-    if featureset.get('globalFeatures').length
-      table.table.table-condensed
-        tr
-          th ID
-          th Name
-          th Type
-          th Values
-        each feature in featureset.get('globalFeatures')
-          tr
-            td= feature.id
-            td= feature.name.join(', ')
-            td= feature.type
-            td
-              table.table.table-condensed
+  each featureType in [{id: 'globalFeatures', name: 'Global Features'}, {id: 'localFeatures', name: 'Local Features'}]
+    table.table.table-condensed.isic-featureset-table
+      tr
+        td.isic-featureset-table-section-header(colspan=2)= featureType.name
+      tr
+        td(colspan=2)
+          if featureset.get(featureType.id).length
+            table.table.table-condensed.isic-featureset-table-features-table
+              each feature in featureset.get(featureType.id)
                 tr
-                  th ID
-                  th Name
-                each valueOption in feature.options
+                  td.isic-featureset-table-feature-header(colspan=3)= feature.name.join(', ')
+                    | :
```

Comment:
I don't think we want this `:`, since the change in font already differentiates the name from the rest.